s3transfer/utils.py

```diff
@@ -126,6 +126,24 @@ def invoke_progress_callbacks(callbacks, bytes_transferred):
             callback(bytes_transferred=bytes_transferred)
 
 
+def get_filtered_extra_args(extra_args, whitelisted_args):
```

Comment:
Can you rename this to something more general?  The name makes sense when it's an internal method on the various Task subclasses, but if it's going to be a utility function I think it should be named without any mention of "extra args".

Reply:
Yeah I was thinking about that too. I am going to rename it to `get_filtered_dict`.